state.go

```diff
@@ -714,6 +714,35 @@ func (s *State) onInterface(se *Session, i interface{}) (err error) {
 		if s.TrackVoice {
 			err = s.voiceStateUpdate(t)
 		}
+	case *PresenceUpdate:
+		if s.TrackMembers {
+			var m *Member
+			m, err = s.Member(t.GuildID, t.User.ID)
```

Comment:
Line 730 currently needs to assign the `err` defined in the function's signature, so that that error can be returned. That could be changed to `return s.MemberAdd(m)` if you'd prefer, and this to `:=`?